app/v2/controllers/ErrorTranslator.scala

```diff
@@ -63,15 +70,17 @@ trait ErrorTranslator {
   implicit val persistenceErrorConverter = new Converter[PersistenceError] {
 
```

Comment:
This failed to compile because you've now got ambiguous imports. You should locally import the persistence errors here:

`import v2.models.errors.PersistenceError._`

and remove the imports in the global scope.